apps/researcher/src/lib/community/signed-in.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
+
+import {SignedIn as ClerkSignedIn} from '@clerk/nextjs';
+import {ReactNode} from 'react';
+
+/*
+Use this component to wrap content that should only be visible to signed in users.
+Do not use the <SingIn /> component from @clerk/nextjs directly,
```

Comment:
Minor: `do not use the <SignedIn /> component...`? Although I do like to sing in :) 🎶🎶